components/GeoApi.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { resolve } from "styled-jsx/css";
+import style from "../styles/Home.module.css";
+
+function GeoApi(props) {
+  const { weatherForecast, setWeatherForecast } = props;
+  const [locationName, setLocationName] = useState("Linn, Kansas");
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleChange = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  async function fetchCoords() {
+    let long = 0;
```

Comment:
I don't think these `let` statements are necessary since their values are changed. I think they could be defined as `const` on line 23 and 24

Reply:
I agree, I'll get rid of the let statements. I also think we should make an .env template file like you suggested. I have a few questions about the template though, I'll ask you about them during our zoom meeting.